<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reconocimiento de iris</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Video</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="IrisVideoWidget" name="video" native="true">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="ImageWidget" name="animacionEnfoque" native="true"/>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnCapturar">
           <property name="text">
            <string>Capturar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnForzarRegistracion">
           <property name="text">
            <string>Forzar registraci칩n</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnForzarIdentificacion">
           <property name="text">
            <string>Forzar Identificaci칩n</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Imagen Capturada</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="ImageWidget" name="capturedImage" native="true">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Acci칩n autom치tica</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="autoIdentify">
              <property name="text">
               <string>Identificar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="automaticaNada">
              <property name="text">
               <string>Ninguna</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
         <zorder></zorder>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="btnIdentificar">
           <property name="text">
            <string>Identificar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRegistrar">
           <property name="text">
            <string>Registrar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnGuardarImagen">
           <property name="text">
            <string>Guardar imagen</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="focusLabel">
           <property name="text">
            <string>Enfoque</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="focusScore">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Calidad Iris</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="irisScore">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QCheckBox" name="chkGuardarVideo">
           <property name="text">
            <string>Guardar video</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="debugWindow">
           <property name="text">
            <string>Ventana de debug</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header>imagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IrisVideoWidget</class>
   <extends>QWidget</extends>
   <header>irisvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
